keps/sig-windows/2258-kubectl-service-log-viewer/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: KEP Template
+kep-number: 2258
+authors:
+  - "@aravindhp"
+  - "@LorbusChris"
+owning-sig: sig-windows
+participating-sigs:
+  - sig-node
+status: provisional
```

Comment:
Hi there, 1.21 Enhancements Lead here
I realized that this is WIP, but wanted to give some initial feedback from the release team's perspective. One of the requirements for a KEP to be included in the release is to have a status of `implementable`. Please make sure that the status is changed before the enhancements freeze. thank you!

Reply:
I have done so @annajung. But we still need to work through some of the other sections. Will do so before the freeze.